python/evaluation_tools/evaluation.py

```diff
@@ -40,10 +40,21 @@ def runEvaluations(self):
             self.logger.info("=== Run Evaluation ===")
             if 'name' in evaluation:
                 if 'package' in evaluation:
-                    package_path = catkin_utils.catkinFindLib(
-                        evaluation['package'])
-                    evaluation_script_with_path = os.path.join(
-                        package_path, evaluation['name'])
+                    evaluation_script_with_path = None
+                    try:
+                        package_path = catkin_utils.catkinFindLib(
+                            evaluation['package'])
+                        evaluation_script_with_path = os.path.join(
+                            package_path, evaluation['name'])
+                    except ValueError:
+                        pass
```

Comment:
print some error message here?

Reply:
no because it's not really an error, it's corrected below